recipes/elpigraph-python/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "elpigraph-python" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 15ac2dcefe4d2a0b475359d10471f0ec2e8c069100dc81208c924efe845231dd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+requirements:
+  host:
+    - python
+    - pip
+    - numpy >=1.16.2
+    - pandas >=0.23.4
+    - numba >=0.49.1 
+    - scikit-learn >=0.21.3
+    - scipy >=1.2.0
+    - python-igraph >=0.7.1
+    - matplotlib
```

Comment:
Are all of these needed to build this package. This is usually just pip & python